lib/urbanopt/scenario/default_reports/end_uses.rb

```diff
@@ -45,26 +45,11 @@ def defaults
     
     end
 
-    def merge_end_uses(existing_period, new_period)
```

Comment:
@rawadelkontar Ruby does not have static type checking, naming and convention is extremely important.  Here you had things named as reporting periods passed into a method that wanted end uses.